packages/cozy-procedures/src/components/Documents.jsx

```diff
@@ -25,12 +59,20 @@ class Documents extends React.Component {
           <LoadingDocumentHolder />
           <DocumentHolder />
         </div>
-        {fields.map((document, index) => (
-          <section key={index}>
-            <Label>{t(`documents.labels.${document.label}`)}</Label>
-            <EmptyDocumentHolder />
-          </section>
-        ))}
+        {Object.values(populatedTemplateDocsWithFiles).map(
+          (document, index) => {
+            const { files, count: templateDocumentsCount } = document
+            return (
+              <section key={index}>
+                <Label>{t(`documents.labels.${document.label}`)}</Label>
+                <DocumentsGroup
+                  files={files}
+                  templateDocumentsCount={templateDocumentsCount}
```

Comment:
This prop name does not seem very clear to me. In the end, I think I guessed that it is the number of docs for this "category". I would at least remove `template`. Or maybe something like `size` (size of the group)